app/com/linkedin/drelephant/spark/fetchers/statusapiv1/statusapiv1.scala

```diff
@@ -87,7 +86,9 @@ trait ExecutorSummary{
   def totalShuffleRead: Long
   def totalShuffleWrite: Long
   def maxMemory: Long
-  def executorLogs: Map[String, String]}
+  def executorLogs: Map[String, String]
+  def peakUnifiedMemory: Map[String, Long]
```

Comment:
How do you get peakUnifiedMemroy value? I don't think Spark history server reports it.